src/components/dialogs/parameters/security-analysis/limit-reductions-table-form.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, {
+    FunctionComponent,
+    useCallback,
+    useEffect,
+    useMemo,
+} from 'react';
+import {
+    COLUMNS_DEFINITIONS_LIMIT_REDUCTIONS,
+    ILimitReductionsByVoltageLevel,
+    IST_FORM,
+    ITemporaryLimitReduction,
+    LIMIT_DURATION_FORM,
+    LIMIT_REDUCTIONS_FORM,
+    VOLTAGE_LEVELS_FORM,
+} from './columns-definitions';
+import { useIntl } from 'react-intl';
+import LimitReductionsTable from './limit-reductions-table';
+import { UseFormReturn } from 'react-hook-form';
+
+const LimitReductionsTableForm: FunctionComponent<{
+    formMethods: UseFormReturn;
+    limits: ILimitReductionsByVoltageLevel[];
```

Comment:
```suggestion
    limits?: ILimitReductionsByVoltageLevel[];
```

I think limit can be undefined, otherwise all "if(limit)" in this file don't make sense
Maybe we should not render this component at all if limit is undefined ?

Reply:
Tests removed